### PR TITLE
New tenant - HPNonline

### PR DESCRIPTION
**Pull request by Alixers505**
https://www.pivotaltracker.com/story/show/172264655

Note -- This is a phase 1. Still waiting on final replies from the stakeholders

### CODE REVIEW

File: packages/common/components/style-b/default-style.marko [hunk 1]
```diff
@@ -14,26 +15,26 @@
...
   .footer-nav {
-    font: normal .75rem/1.5 Arial, sans-serif;
+    font: normal 14px/1.5 Arial, sans-serif;
     padding: 5px 10px;
   }
 
   .body h3,
   .body h3 a {
-    color: #000000;
-    font: normal 28px/1.5 Arial, Helvetica, sans-serif;
```
Comment (CaitlynKnudsvig) [inline packages/common/components/style-b/default-style.marko:24]: Why is Helvetica being removed?  Apple mail still uses a default font afaik?  Also, might as well update the line heights to px here so it's one less thing to have to fix later.

File: tenants/hpnonline/templates/daily.marko [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+          <common-table width="700" style="border-collapse:collapse;" align="center" class="main" padding=0 spacing=0>
+            <tr>
+              <td width="300" align="left">
+                <common-advertisement-element
+                  name=mrPrimary
+                />
+              </td>
+              <td width="300" align="right">
+                <common-advertisement-element
+                  name=mrSecondary
```
Comment (CaitlynKnudsvig) [inline tenants/hpnonline/templates/daily.marko:73]: These will likely need `date=date` along with the leaderboards

File: packages/common/components/style-b/default-style.marko (hunk 1 above)
Comment (Alixers505) [reply packages/common/components/style-b/default-style.marko:24]: the px are corrected for the style-b components in a diff PR

File: tenants/hpnonline/templates/daily.marko [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+          <common-table width="700" style=titleTableStyle align="center" class="main" padding=0 spacing=0>
+            <tr>
+              <td>
+                <h3 style=`${titleStyle}`>Table of Contents</h3>
+              </td>
+            </tr>
+          </common-table>
+
+          <common-leaderboard-primary-block
+            name=leaderboardPrimary
```
Comment (CaitlynKnudsvig) [inline tenants/hpnonline/templates/daily.marko:53]: `leaderboardPrimary` and `leaderboardSecondary` in this file should also have `date=date`, the rest looks good.

Comment (Alixers505) [reply tenants/hpnonline/templates/daily.marko:53]: The joy of not having placeholder ads we can reference for new setups :(

**Pull request merged**